website/docs/feature-flag-tutorials/react/examples.md

```diff
@@ -283,16 +283,16 @@ Let’s walk through how to enable impression data for the feature flag we creat
 
 ### Enable feature flag impression data
 
-At the flag level in Unleash, navigate to the [Settings view](http://localhost:4242/projects/default/features/newNotificationsBadge/settings).
+At the flag level in Unleash, navigate to the Settings view.
 
 
-![Edit Settings image](/img/react-ex-edit-settings.png)
+![Image of editing flag Settings in Unleash](/img/react-ex-edit-settings.png)
```

Comment:
Screen readers already announce that it's an image before reading the alt text, so you don't need to specify "image of" in alt text. Here is a nice [write-up about alt texts](https://axesslab.com/alt-texts/) by a vision-impaired developer who uses a screen-reader in his day-to-day. It mentions avoiding starting the text with "image of", plus many more things (such as finishing with full stops).

```suggestion
![Editing feature flag settings in Unleash.](/img/react-ex-edit-settings.png)
```